solr/modules/ltr/example/train_and_upload_demo_model.py

```diff
@@ -8,8 +8,15 @@
 
 from optparse import OptionParser
 
-solrQueryUrl = ""
+import solr
+from solr.api import querying_api
 
+def setupSolrClient(host, port):
+    '''Configures the Solr client with the specified Solr host/port'''
+    solr_client_config = solr.Configuration(
+            host = "http://" + host + ":" + str(port) + "/api"
```

Comment:
i guess at some point the /api will get dropped right?

Reply:
I think that's the plan eventually, yep.